doc/ug/dv_methodology/index.md

```diff
@@ -548,12 +549,33 @@ For example, when a_valid is 0 in the TLUL interface, we drive data, address and
 
 The simulator may report that some portions of RTL / DV could not be instrumented for X-propagation due to the way they were written.
 This is typically captured in a log file during the simulator's build step.
-It is mandatory to analyze these pieces of logic to either refactor them to make them more amenable to X-prop instrumentation, or waive them if it is not possible to do so.
+It is mandatory to analyze these pieces of logic to either refactor them to be amenable to X-prop instrumentation, or waive them if it is not possible to do so.
 This is captured as the V3 checklist item `X_PROP_ANALYSIS_COMPLETED`.
 
+### Common XProp Issues
+
+Xprop instrumentation is simulator specific; this focuses on VCS, and lists some simple and common constructs that block instrumentation.
+
+    *  Some behavioral statements, like `break`, `continue`, `return`, and `disable`.
+    *  Functions that have multiple `return` statements.
+    *  Calling tasks or functions with side-effects, notably using the `uvm_info` macro.
```

Comment:
is `real` type also not instrumentable  for VCS? If so, we can include it as well.

Reply:
I see. Thanks